Automattic/simplenote-ios
# Fix: Share Extension Navigation Tint

### Description
In this PR we're fixing a bug that caused the Share Extension navigation buttons not to be visible.

@bummytime you game for an old school fix? 😄 
Thanks in advance sir!

Closes #773

### Details
The root of this issue was the Host App setting `UIBarButtonItem.appearance`, which was inherited by the Share Extension UI.

Plus: Since we were there... we've nuked dead code, and simplified the Appearance initialization.

### Test
1. Switch to Light Mode
2. Launch WordPress iOS and open the Reader
3. Open any post and press over the top right corner, to initiate Share

- [x] Verify the Cancel / Save buttons (in the Share Extension's Navigation) are visible
- [x] Please repeat for iOS 

### Release
`RELEASE-NOTES.txt` was updated in d577df0 with:
 
> Fixed a bug that affected the Share Extension's Appearance.
